crates/nargo/src/cli/compile_cmd.rs

```diff
@@ -19,7 +19,7 @@ pub(crate) fn run(args: ArgMatches) -> Result<(), CliError> {
 
     let curr_dir = std::env::current_dir().unwrap();
     let mut circuit_path = PathBuf::new();
-    circuit_path.push(BUILD_DIR);
+    circuit_path.push(CHECK_DIR);
```

Comment:
This is the path to change the output of the `compile` command, I think we should rename `CHECK_DIR` back to `BUILD_DIR` or to `COMPILE_CMD_DIR` to be more clear. The path should also be changed back to `build` or `target` for this.

Reply:
makes sense